pkg/agent/controller/traceflow/packetin.go

```diff
@@ -222,6 +226,10 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1beta1.Traceflo
 		npRef := c.networkPolicyQuerier.GetNetworkPolicyByRuleFlowID(ingressInfo)
 		if npRef != nil {
 			ob.NetworkPolicy = npRef.ToString()
+			ruleRef := c.networkPolicyQuerier.GetRuleByFlowID(ingressInfo)
```

Comment:
Could you check and update the unit test?

Reply:
updated, PTAL.
Thanks.